webpack/assets/javascripts/react_app/components/notifications/toast/Toast.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import Alert from '../../common/Alert';
+
+const Toast = (props) => {
+  return (
+    <Alert
+      type={props.type}
+      dismissable={props.close}
+      css="toast-pf"
+      >
+      <div className="pull-right toast-pf-action">
+        <a href="#">
+          {props.link}
```

Comment:
building the JSX such that the <a> is only include when there actually is a link. pointer cursor not desirable when there is nothing to click.